docs/api/covidcast-signals/hhs.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Department of Health & Human Services
+parent: Data Sources and Signals
+grand_parent: COVIDcast Epidata API
+---
+
+# SOURCE NAME
+{: .no_toc}
+
+* **Source name:** `hhs`
+* **First issued:** TBD
+* **Number of data revisions since 19 May 2020:** 0
+* **Date of last change:** Never
+* **Available for:** county, hrr, msa, state (see [geography coding docs](../covidcast_geography.md))
+* **Time type:** day (see [date format docs](../covidcast_times.md))
+* **License:** [Open Data Commons Open Database License (ODbL)](https://opendatacommons.org/licenses/odbl/1-0/)
+
+The U.S. Department of Health & Human Services (HHS) publishes several
+datasets on patient impact and hospital capacity. Two of these
+datasets are mirrored in Epidata at the following endpoints:
+
+* [COVID-19 Hospitalization: States](../covid_hosp.md) - daily resolution, state aggregates
+* [COVID-19 Hospitalization: Facilities](../covid_hosp_facilities.md) - weekly resolution, individual hospitals
+
+Both datasets contain dozens of columns that break down hospital
+resource usage in different ways.
+
+This indicator makes available several commonly-used combinations of
+those columns, aggregated geographically. In particular, we include
+the sum of all adult and pediatric COVID-19 hospital admissions. This
+sum is used as the "ground truth" for hospitalizations by the [Reich
+lab COVID-19 forecast
+hub](https://github.com/reichlab/covid19-forecast-hub/blob/master/data-processed/README.md#hospitalizations).
```

Comment:
@RoniRos is this still the case in light of the coverage/reporting issues?

Reply:
For state, yes. [Update in slack](https://delphi-org.slack.com/archives/C01GJCMFTN1/p1610634088000100?thread_ts=1610383099.021800&cid=C01GJCMFTN1)

Reply:
:+1: